dev/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Request from 'superagent';
+
+
+export default class App extends React.Component {
+	
+	constructor(){
+		super();
+	}
+
+	componentWillMount() {
+		var url = "http://localhost:8088/services/users/all";
```

Comment:
Here should be something like **usersResource.getAllUsers()**
All the internal request information should be moved to a separate file